browser/mapping.go

```diff
@@ -52,8 +53,14 @@ func mapLocator(vu moduleVU, lo *common.Locator) mapping {
 			return lo.Clear(copts) //nolint:wrapcheck
 		},
 		"click": func(opts goja.Value) *goja.Promise {
+			popts, err := parseFrameClickOptions(vu.Context(), opts, lo.Timeout())
+
 			return k6ext.Promise(vu.Context(), func() (any, error) {
-				err := lo.Click(opts)
+				if err != nil {
+					return nil, err
+				}
+
+				err = lo.Click(popts)
 				return nil, err //nolint:wrapcheck
```

Comment:
Is there a functional reason for assigning this operation's result to `err` here?

Reply:
You mean, why it is not:
```suggestion
				return nil, lo.Click(popts) //nolint:wrapcheck
```
? :+1: (nit)

Reply:
Changed in eddc74e79501e35676b25af76f6193dffefd3a1d. I've also returned early from any parse errors in the same commit.